Hometask_Lec7/Task1.cs

```diff
@@ -1,2 +1,134 @@
+
+    //        Console.WriteLine(Goober.AnimalData);                                                     // return the edited data of dolphin Goober
+    //        Console.WriteLine(Murzik.AnimalData);                                                     // return the edited data of cat Mursik
+
+    //        Dolphin.Delimiter();                                                                      // added a delimiter to make the output data more readable
+
+    //        Console.WriteLine(Dolphin.Feed(Goober.Nickname, Goober.isHungry));                        // Feeded dolphin Goober
+    //        Console.WriteLine(Cat.Feed(Murzik.Nickname, Murzik.isHungry));                            // Feeded cat Murzik
+
+    //        Dolphin.Delimiter();                                                                      // added a delimiter to make the output data more readable
+
+    //        Console.WriteLine(Goober.SayHelloToTheAnimal(Goober.Nickname));                           // saying hello to Goober
+    //        Console.WriteLine(Murzik.SayHelloToTheAnimal(Murzik.Nickname));                           // saying hello to Murzik
+
+    //        Dolphin.Delimiter();                                                                      // added a delimiter to make the output data more readable
+
+    //        Console.WriteLine(Dolphin.Swim(Goober.Nickname));                                         // Dolphin Goober is swimmig
+    //        Console.WriteLine(Cat.CatIsShedding(Murzik.Nickname));                                    // Мурзик линяет
+
+    //        Dolphin.Delimiter();                                                                      // added a delimiter to make the output data more readable
+    //    }
+    //}
+
+    public abstract class Mammal                                                 // млекопитающие
+    {
+        public string KindOfAnimal;                                              // fields of the abstract class
+
+        public string Nickname;
+        public int Age;
+        public bool isHungry;
+
+        public Mammal(string kindOfAnimal, string nickname, int age, bool isHungry)        // set initial object values data
+        {
+            this.KindOfAnimal = kindOfAnimal;
+            this.Nickname = nickname;
+            this.Age = age;
+            this.isHungry = isHungry;
+        }
+
+        public abstract string AnimalData { get; set; }                            // abstract getter and setter
+
+        public static string Feed(string petName, bool isHungry)                   // feed pet if the pet is hungry
+        {
+            if (isHungry)
+            {
+                return $"{petName} is feeded!";
+            }
+            else { return $"{petName} is not hungry."; }
+        }
+
+        public static void Delimiter()                                            // methot adds a delimiter to make the output data more readable
```

Comment:
I think it's better to create a method that wtites the animal data to console and use the delimeter after every call. There's not gonna be any method like this. It does not have any connection to mammal

Reply:
deleted Delimiter()  method from the abstract class and Delimiter() method calls fron Main() method.
Added delimiters after animal data output directly in Main() method.